android-release-build.md

```diff
@@ -139,6 +139,54 @@ file.
 
 Release builds of your app will now automatically be signed.
 
+
+## Minfify and obfuscate
+
+By default, Flutter does not obfuscate or minify Android wrapper. If you intend to use third-party Java or Android libraries, you may want to reduce the size of the APK or protect that code from reverse engineering.
+
+### Step 1 - Configure obfuscation tool
+
+Create `/android/app/proguard-rules.pro` file and add rules listed below.
+
+```
+#Flutter Wrapper
```

Comment:
Should this just be part of our template?  @tvolkert @mravn-google ?

Reply:
@eseidelGoogle this was the initial idea, however I agree that obfuscation and shrinking is only needed in specific scenarios e.g. hybrid app with large libraries, non-feature requirements etc.

Also running ProGuard slows down build time, which maybe not the big issue for Flutter developers but has it's impact on CI build times.